.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Bug report
+description: Report a bug or an issue you've found with Trino Python client
+labels: bug
+body:
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What do you think should have happened
+      placeholder: >
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      description: Describe what actually happened
+      placeholder: >
+        A clear and concise description of what actually happened.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: This will help us reproduce your issue
+      placeholder: >
+        In as much detail as possible, please provide steps to reproduce the issue.
+        Sample code that triggers the issue, relevant server settings, etc is all very helpful here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots and log output
+      description: What do you think went wrong?
+      placeholder: >
+        If applicable, add screenshots or log output to help explain your problem.
+  - type: input
+    attributes:
+      label: Operating System
+      description: What Operating System are you using?
+      placeholder: "You can get it via `cat /etc/os-release` for example"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Trino Python client version
+      description: Which version of Trino Python client are you using?
+      placeholder: "Execute `python3 -c 'import trino; print(trino.__version__)'`"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Trino Server version
+      description: Which Trino server version are you using?
+      placeholder: "Execute `SELECT VERSION();` on your Trino server"
```

Comment:
No changes requested, "Execute ... on your Trino **_server_**" sounds little weird to me.

Reply:
Maybe "Run" is slightly better?